bundler/spec/commands/install_spec.rb

```diff
@@ -29,6 +29,28 @@
       expect(bundled_app_lock).to exist
     end
 
+    it "creates lockfile based on the lockfile method in Gemfile" do
```

Comment:
Should we make a warning when the user has a `Gemfile` & `Gemfile.lock`, but runs `bundle install` with `--no-lock` ?

Reply:
Actually I've also found that when you run `--no-lock` with an existing `Gemfile` & `Gemfile.lock`, Bundler will load the existing `Gemfile.lock`. I think this _shouldn't_ be happening in this situation, but I'm keen to hear thoughts?